frontend/src/components/forms/CreateEmployee.tsx

```diff
@@ -0,0 +1,320 @@
+            pr="16px"
+            mt="10px"
+            mb="10px"
+            ml="10px"
+            mr="10px"
+            borderBottom="1px solid #EBECF0"
+          >
+            <ModalHeader>Add Employee</ModalHeader>
+            <ModalCloseButton mt="5px" margin="inherit" size="lg" />
+          </Box>
+          <ModalBody>
+            <Box pt="7px" pb="7px" pl="16px" pr="16px">
+              <Box
+                display="flex"
+                flexDirection="row"
+                alignItems="flex-start"
+                justifyContent="space-between"
+              >
+                <Box w="46%">
+                  <FormControl isRequired isInvalid={invitedFirstNameError}>
+                    <FormLabel>First Name</FormLabel>
+                    <Input
+                      placeholder="Enter first name"
+                      value={invitedFirstName}
+                      onChange={handleFirstNameChange}
+                      maxLength={50}
+                    />
+                    <FormErrorMessage>First Name is required.</FormErrorMessage>
+                  </FormControl>
+                </Box>
+
+                <Box w="46%">
+                  <FormControl isRequired isInvalid={invitedLastNameError}>
+                    <FormLabel>Last Name</FormLabel>
+                    <Input
+                      placeholder="Enter last name"
+                      value={invitedLastName}
+                      onChange={handleLastNameChange}
+                      maxLength={50}
+                    />
+                    <FormErrorMessage>Last Name is required.</FormErrorMessage>
+                  </FormControl>
+                </Box>
+              </Box>
+              <Box display="flex" flexDirection="row" mt="16px">
+                <FormControl isRequired isInvalid={invitedEmailError}>
+                  <FormLabel>SHOW Email</FormLabel>
+                  <Input
+                    placeholder="Enter SHOW email"
+                    value={invitedEmail}
+                    onChange={(event) => setInvitedEmail(event.target.value)}
+                    maxLength={254}
+                  />
+                  <FormErrorMessage>
+                    A validwh email is required.
+                  </FormErrorMessage>
+                </FormControl>
+              </Box>
+            </Box>
+            <Box
+              pt="17px"
+              pb="7px"
+              pl="24px"
+              pr="24px"
+              borderBottom="1px solid #EBECF0"
+            >
+              <FormControl isRequired isInvalid={invitedAdminStatusError}>
+                <RadioGroup
+                  value={invitedAdminStatus}
+                  onChange={handleAdminStatusChange}
+                  pb="24px"
+                >
+                  <Flex>
+                    <Box marginRight="24px">
+                      <Radio size="lg" colorScheme="teal" value="1">
+                        <Text color="#1B2A2C">Admin</Text>
```

Comment:
I could not figure out what is causing the font difference, if you want a band-aid fix `fontWeight="medium"` makes the text look more like the input text. I don't think the font difference is a huge issue though.

This is what it looks like using `<Text fontWeight="medium" color="#1B2A2C"></Text>`:
![image](https://github.com/uwblueprint/supportive-housing/assets/43042601/4b3a1884-cc0b-4574-a5f3-9652f0d2d4f5)